content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -166,14 +166,37 @@ If you keep the entry criteria as "All Contacts" it's important to remember that
 
 ## Email Testing
 
-Email Testing adds spam testing, email rendering, and link validation directly in the Automation experience so you can see what your emails will look like across a variety of platforms and devices.
+Email Testing adds spam testing, email rendering, and link validation directly in the Automation experience so you can see what your emails will look like across a variety of devices, email clients, and inbox providers. 
```

Comment:
"Email testing adds spam testing, email rendering previews, and link validation natively into the Automation experience. This testing allows you to preview how your email is likely to perform across a wide range of inbox providers, devices and spam filters."